src/components/side-by-side/side-by-side.tsx

```diff
@@ -53,7 +58,7 @@ export class SideBySide extends Component<SideBySideComponentProps> {
           <Button
             className={styles.iconContainer}
             onClick={onExpand}
-            tooltip={{label: this.props.expandScreen!, type: 'bottom-left'}}
+            tooltip={{label: this.props.streamMode === StreamMode.Primary ? this.props.expandPrimaryScreen! : this.props.expandSecondaryScreen!, type: 'bottom-left'}}
```

Comment:
I would recommend move the condition inside translates.
Example: https://github.com/kaltura/playkit-js-navigation/blob/master/src/components/navigation/plugin-button/index.tsx#L10-L14